src/Component/Avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Pls investigate why do we have to import this. Created a ticket for this.
https://richmondcroft.atlassian.net/browse/CHAT-42